prow/jobs/incubator/compass/components/director/director-generic.yaml

```diff
@@ -0,0 +1,49 @@
+# Code generated by rendertemplates. DO NOT EDIT. 
+
+
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+job_template: &job_template
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
```

Comment:
It seems to me as a bug in the generated yaml, this should be `path_alias: github.com/kyma-incubator/compass`